SimpleJavaProject/src/main/java/cristina_savrin/NumbersMain.java

```diff
@@ -0,0 +1,55 @@
+package cristina_savrin;
+
+import java.util.Scanner;
+
+public class NumbersMain {
+
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Enter a number: ");
+        int value = scanner.nextInt();
+
+        // 1. Find the square root of a number.
+        System.out.println("The square root of " + value + " is: " + Math.sqrt(value));
+
+        // 2. Generate a random number between 0 and 100. (Try the same between 10 and 100)
+        // Formula: (Math.random() * (max - min) + min)
+        System.out.println("\nRandom number between 0 and 100: " + (int) (Math.random() * 100));
+        System.out.println("Random number between 10 and 100: " + (int) (Math.random() * (100 - 10) + 10));
```

Comment:
100 - 10 = 90 you don't need to calculate it each time

Reply:
Corrected. Thank you!